krypto/src/commonMain/kotlin/com/soywiz/krypto/encoding/Base64.kt

```diff
@@ -8,6 +8,8 @@ object Base64 {
             this[TABLE[n].code] = n
         }
     }
+    private val TABLE_URL = TABLE.toUrlEncode()
+    private val DECODE_URL = DECODE.toUrlDecode()
```

Comment:
I believe `toUrlEncode` and `toUrlDecode` functions would add more payload size than providing a literal or an inline replacement.

Maybe we can either provide `TABLE_URL` as a literal, or something like `TABLE_URL = TABLE.replace().replace()`.
Since the code for constructing the `DECODE` table is still required, maybe we can extract that to a function (receiving `TABLE` as parameter), and use it for both `DECODE` and `DECODE_URL`.

What do you think?

Reply:
Other than that, I really like what you did here. Storing the right table at the beginning keeps the rest of the code intact, so it is super nice.

Reply:
I wouldn't expect defining the function as inline vs a separate function to have much impact on the final compiled code size. But I also realized that using `replace()` twice means an extra String array is going to be allocated in memory, and then garbage collected. Plus there is the initialization execution time required when first referencing the Base64 object. We're talking about small array sizes here so it's probably not a big deal, but for the sake of having code uniformity, I made `TABLE_URL` a literal to mirror the `TABLE` literal.

For `DECODE_URL`, there is an advantage to reusing `DECODE`, namely only having to copy it and then modify 2 elements. That should be faster than regenerating a copy directly from the TABLE_URL array. But again the size here is really small so it's probably more important to reduce the amount of code and make it maintainable. So I extracted a function here to be used for both DECODE arrays.

Good suggestions.